### PR TITLE
started implementing the design

### PR DESCRIPTION
**Pull request by mushon**

### CODE REVIEW

File: app/styles/main.scss [hunk 1]
```diff
@@ -14,16 +20,34 @@ html, body, .leaflet-container, .container {
...
+    display: block;
+    height: 117px; width: 128px;
+  }
+  
 }
 
-.leaflet-div-icon {
-  padding: 20px;
+div.leaflet-div-icon {
+  //padding: 20px;
```
Comment (amitaibu) [inline app/styles/main.scss]: Please remove commented out code.

File: app/styles/main.scss [hunk 2]
```diff
@@ -14,16 +20,34 @@ html, body, .leaflet-container, .container {
...
+  border: 0;
+  background: url(../images/marker@2x.png) no-repeat;
+  display: block;
+  height: 59px; width: 71px;
+  .marker-icon{
```
Comment (amitaibu) [inline app/styles/main.scss]: Should be `.marker-icon {` (with space).

Comment (amitaibu) [inline app/styles/main.scss]: Also add a line break before to separate it from the parent

File: app/styles/main.scss [hunk 3]
```diff
@@ -14,16 +20,34 @@ html, body, .leaflet-container, .container {
...
 .add-marker-wrapper {
   position: absolute;
-  top: 40px;
-  right: 40px;
   z-index: 99;
-  padding: 20px 10px;
-  background: black;
-  color: white;
+  //padding: 20px 10px;
+  a{
```
Comment (amitaibu) [inline app/styles/main.scss]: missing space (`a {`)

**Comment by mushon** [comment]
I addressed @amitaibu issues in this latest push

File: app/styles/main.scss [hunk 4]
```diff
@@ -14,16 +20,33 @@ html, body, .leaflet-container, .container {
...
   z-index: 99;
-  padding: 20px 10px;
-  background: black;
-  color: white;
+  
+  a {
+    background: url(../images/Add_btn@2x.png) no-repeat;
+    text-indent: -99999px;
+    display: block;
+    height: 117px; width: 128px;
```
Comment (amitaibu) [inline app/styles/main.scss]: ok, last nit-pick, every declaration should be in own line. (here, and below)

**Comment by amitaibu** [comment]
10x, Merged. Please open a new PR for other pages (i.e. after merge, the branch is not used)

**Pull request closed without merging**